tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -519,6 +519,16 @@ class WskBasicUsageTests
             }
     }
 
+    it should "invoke an action implicit blocking when specifying --result" in withAssetCleaner(wskprops) {
```

Comment:
I would actually flip this so that it tests that you can still pass --blocking and --result and change the other tests to use just --result:

in `tests/src/test/scala/system/basic/WskBasicTests.scala`
```
wsk.action.invoke(name, parameters = input, blocking = true, result = true, expectedExitCode = 246).
wsk.action.invoke(name, Map("payload" -> "one two three".toJson), blocking = true, result = true)
val stdout = wsk.action.invoke(name, blocking = true, result = true).stdout
action.invoke(name, parameters = params, blocking = true, result = true, expectedExitCode = ACCEPTED)
```